<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>PPSSPP</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QtEmuGL" name="widget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_FileLoad"/>
    <addaction name="action_FileClose"/>
    <addaction name="separator"/>
    <addaction name="action_FileQuickloadState"/>
    <addaction name="action_FileQuickSaveState"/>
    <addaction name="action_FileLoadStateFile"/>
    <addaction name="action_FileSaveStateFile"/>
    <addaction name="separator"/>
    <addaction name="action_FileExit"/>
   </widget>
   <widget class="QMenu" name="menu_Emulation">
    <property name="title">
     <string>&amp;Emulation</string>
    </property>
    <addaction name="action_EmulationRun"/>
    <addaction name="action_EmulationPause"/>
    <addaction name="separator"/>
    <addaction name="action_EmulationReset"/>
    <addaction name="separator"/>
    <addaction name="action_EmulationRunLoad"/>
   </widget>
   <widget class="QMenu" name="menuDebu_g">
    <property name="title">
     <string>Debu&amp;g</string>
    </property>
    <addaction name="action_DebugLoadMapFile"/>
    <addaction name="action_DebugSaveMapFile"/>
    <addaction name="action_DebugResetSymbolTable"/>
    <addaction name="separator"/>
    <addaction name="action_DebugDumpFrame"/>
    <addaction name="separator"/>
    <addaction name="action_DebugDisassembly"/>
    <addaction name="action_DebugDisplayList"/>
    <addaction name="action_DebugLog"/>
    <addaction name="action_DebugMemoryView"/>
    <addaction name="action_DebugMemoryViewTexture"/>
   </widget>
   <widget class="QMenu" name="menu_Options">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <widget class="QMenu" name="menu_Log_Levels">
     <property name="title">
      <string>Lo&amp;g Levels</string>
     </property>
     <widget class="QMenu" name="menuG3D">
      <property name="title">
       <string>G3D</string>
      </property>
      <addaction name="actionLogG3DDebug"/>
      <addaction name="actionLogG3DWarning"/>
      <addaction name="actionLogG3DInfo"/>
      <addaction name="actionLogG3DError"/>
     </widget>
     <widget class="QMenu" name="menuHLE">
      <property name="title">
       <string>HLE</string>
      </property>
      <addaction name="actionLogHLEDebug"/>
      <addaction name="actionLogHLEWarning"/>
      <addaction name="actionLogHLEInfo"/>
      <addaction name="actionLogHLEError"/>
     </widget>
     <widget class="QMenu" name="menuDefault">
      <property name="title">
       <string>Default</string>
      </property>
      <addaction name="actionLogDefDebug"/>
      <addaction name="actionLogDefWarning"/>
      <addaction name="actionLogDefInfo"/>
      <addaction name="actionLogDefError"/>
     </widget>
     <addaction name="menuDefault"/>
     <addaction name="separator"/>
     <addaction name="menuG3D"/>
     <addaction name="menuHLE"/>
    </widget>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>&amp;Language</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuVideo">
     <property name="title">
      <string>&amp;Video</string>
     </property>
     <widget class="QMenu" name="menuAnisotropic_filtering">
      <property name="title">
       <string>&amp;Anisotropic filtering</string>
      </property>
      <addaction name="action_AFOff"/>
      <addaction name="action_AF2x"/>
      <addaction name="action_AF4x"/>
      <addaction name="action_AF8x"/>
      <addaction name="action_AF16x"/>
     </widget>
     <widget class="QMenu" name="menuZoom">
      <property name="title">
       <string>&amp;Zoom</string>
      </property>
      <addaction name="action_OptionsScreen1x"/>
      <addaction name="action_OptionsScreen2x"/>
      <addaction name="action_OptionsScreen3x"/>
      <addaction name="action_OptionsScreen4x"/>
     </widget>
     <addaction name="menuAnisotropic_filtering"/>
     <addaction name="action_OptionsBufferedRendering"/>
     <addaction name="action_OptionsLinearFiltering"/>
     <addaction name="action_Simple_2xAA"/>
     <addaction name="separator"/>
     <addaction name="menuZoom"/>
     <addaction name="action_Stretch_to_display"/>
     <addaction name="separator"/>
     <addaction name="action_OptionsHardwareTransform"/>
     <addaction name="action_OptionsUseVBO"/>
     <addaction name="action_OptionsVertexCache"/>
     <addaction name="action_OptionsDisplayRawFramebuffer"/>
     <addaction name="actionFrameskip"/>
    </widget>
    <widget class="QMenu" name="menuControls">
     <property name="title">
      <string>Co&amp;ntrols</string>
     </property>
     <addaction name="action_OptionsControls"/>
     <addaction name="action_OptionsGamePadControls"/>
    </widget>
    <widget class="QMenu" name="menuCore">
     <property name="title">
      <string>&amp;Core</string>
     </property>
     <addaction name="action_CPUInterpreter"/>
     <addaction name="action_CPUDynarec"/>
     <addaction name="separator"/>
     <addaction name="action_OptionsFastMemory"/>
     <addaction name="action_OptionsIgnoreIllegalReadsWrites"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="menuCore"/>
    <addaction name="menuControls"/>
    <addaction name="menuVideo"/>
    <addaction name="action_Sound"/>
    <addaction name="separator"/>
    <addaction name="action_OptionsFullScreen"/>
    <addaction name="action_OptionsShowDebugStatistics"/>
    <addaction name="action_Show_FPS_counter"/>
    <addaction name="separator"/>
    <addaction name="menu_Log_Levels"/>
    <addaction name="separator"/>
    <addaction name="menuLanguage"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_HelpOpenWebsite"/>
    <addaction name="separator"/>
    <addaction name="action_HelpAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Emulation"/>
   <addaction name="menuDebu_g"/>
   <addaction name="menu_Options"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_FileLoad">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="action_FileClose">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>-</string>
   </property>
  </action>
  <action name="action_FileQuickloadState">
   <property name="text">
    <string>Quickload state</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="action_FileQuickSaveState">
   <property name="text">
    <string>Quicksave state</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="action_FileLoadStateFile">
   <property name="text">
    <string>&amp;Load State File...</string>
   </property>
  </action>
  <action name="action_FileSaveStateFile">
   <property name="text">
    <string>&amp;Save State File...</string>
   </property>
  </action>
  <action name="action_FileExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_EmulationRun">
   <property name="text">
    <string>&amp;Run</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="action_EmulationPause">
   <property name="text">
    <string>&amp;Pause</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="action_EmulationReset">
   <property name="text">
    <string>R&amp;eset</string>
   </property>
  </action>
  <action name="action_CPUInterpreter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Interpreter</string>
   </property>
  </action>
  <action name="action_CPUDynarec">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Dynarec</string>
   </property>
  </action>
  <action name="action_DebugLoadMapFile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Load &amp;Map File...</string>
   </property>
  </action>
  <action name="action_DebugSaveMapFile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Save Map File...</string>
   </property>
  </action>
  <action name="action_DebugResetSymbolTable">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Reset Symbol Table</string>
   </property>
  </action>
  <action name="action_DebugDisassembly">
   <property name="text">
    <string>&amp;Disassembly</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="action_DebugLog">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Log Console</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="action_DebugMemoryView">
   <property name="text">
    <string>Memory &amp;View...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="action_OptionsControls">
   <property name="text">
    <string>&amp;Keyboard</string>
   </property>
  </action>
  <action name="action_OptionsFullScreen">
   <property name="text">
    <string>&amp;Toggle fullscreen</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="action_OptionsBufferedRendering">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Buffered Rendering</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="action_OptionsHardwareTransform">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Hardware Transform</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="action_OptionsLinearFiltering">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Linear Filtering</string>
   </property>
  </action>
  <action name="action_OptionsDisplayRawFramebuffer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Display Raw Framebuffer</string>
   </property>
  </action>
  <action name="action_OptionsShowDebugStatistics">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;debug statistics</string>
   </property>
  </action>
  <action name="action_OptionsScreen1x">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Screen &amp;1x</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="action_OptionsScreen2x">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Screen &amp;2x</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="action_OptionsScreen3x">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Screen &amp;3x</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="action_OptionsScreen4x">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Screen &amp;4x</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
  </action>
  <action name="action_OptionsFastMemory">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Fast Memory (dynarec, unstable)</string>
   </property>
  </action>
  <action name="action_OptionsIgnoreIllegalReadsWrites">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>I&amp;gnore illegal reads/writes</string>
   </property>
  </action>
  <action name="action_HelpOpenWebsite">
   <property name="text">
    <string>&amp;Go to http://www.ppsspp.org/</string>
   </property>
  </action>
  <action name="action_HelpAbout">
   <property name="text">
    <string>&amp;About PPSSPP...</string>
   </property>
  </action>
  <action name="action_OptionsUseVBO">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Use VBO</string>
   </property>
  </action>
  <action name="actionLogG3DDebug">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Debug</string>
   </property>
  </action>
  <action name="actionLogG3DWarning">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Warning</string>
   </property>
  </action>
  <action name="actionLogG3DError">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Error</string>
   </property>
  </action>
  <action name="actionLogG3DInfo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="actionLogHLEDebug">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Debug</string>
   </property>
  </action>
  <action name="actionLogHLEWarning">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Warning</string>
   </property>
  </action>
  <action name="actionLogHLEInfo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="actionLogHLEError">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Error</string>
   </property>
  </action>
  <action name="actionLogDefDebug">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Debug</string>
   </property>
  </action>
  <action name="actionLogDefWarning">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Warning</string>
   </property>
  </action>
  <action name="actionLogDefInfo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="actionLogDefError">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Error</string>
   </property>
  </action>
  <action name="action_OptionsGamePadControls">
   <property name="text">
    <string>&amp;Gamepad</string>
   </property>
  </action>
  <action name="action_EmulationRunLoad">
   <property name="text">
    <string>Run on loa&amp;d</string>
   </property>
  </action>
  <action name="action_DebugDumpFrame">
   <property name="text">
    <string>D&amp;ump next frame to log</string>
   </property>
  </action>
  <action name="action_OptionsVertexCache">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Vertex Cache</string>
   </property>
  </action>
  <action name="action_DebugMemoryViewTexture">
   <property name="text">
    <string>Memory View Texture...</string>
   </property>
  </action>
  <action name="action_DebugDisplayList">
   <property name="text">
    <string>DisplayList...</string>
   </property>
  </action>
  <action name="action_Simple_2xAA">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Simple 2xAA</string>
   </property>
  </action>
  <action name="action_AFOff">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Off</string>
   </property>
  </action>
  <action name="action_AF2x">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2x</string>
   </property>
  </action>
  <action name="action_AF4x">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>4x</string>
   </property>
  </action>
  <action name="action_AF8x">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>8x</string>
   </property>
  </action>
  <action name="action_AF16x">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>16x</string>
   </property>
  </action>
  <action name="action_Show_FPS_counter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;FPS counter</string>
   </property>
  </action>
  <action name="action_Stretch_to_display">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>S&amp;tretch to display</string>
   </property>
  </action>
  <action name="action_Sound">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Sound emulation</string>
   </property>
  </action>
  <action name="actionFrameskip">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Frameskip</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtEmuGL</class>
   <extends>QWidget</extends>
   <header>qtemugl.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
